design/baremetal-operator/reboot-interface.md

```diff
@@ -7,6 +7,8 @@
 
 # reboot-interface
 
+<!-- cSpell:ignore rebalancing,accomodate,unprovisioned,suffixless -->
```

Comment:
accomodate looks wrong to me

Reply:
You mean it should not be ignored, or there is some other spelling for it?

Reply:
Ah, it can have double m.  Fixed.